src/sagemaker/jumpstart/curated_hub/types.py

```diff
@@ -12,11 +12,26 @@
 # language governing permissions and limitations under the License.
 """This module stores types related to SageMaker JumpStart CuratedHub."""
 from __future__ import absolute_import
-from typing import Optional
+from enum import Enum
+from typing import Any, Dict, List, Optional
 
 from sagemaker.jumpstart.types import JumpStartDataHolderType
 
 
+class HubContentType(str, Enum):
```

Comment:
this is confusing, i feel like `HubContentType.Hub` doesn't make sense, it should be in its own category

Reply:
Fair callout, we can make a separate enum to track this